### PR TITLE
cleans up style on pin modal &  normalizes spacing

### PR DESCRIPTION
**Pull request by arkoar**

### CODE REVIEW

**Comment by arkoar** [comment]
This PR cleans up the UI alignment and Spacing on the PIN modals views. A bold and sligtly transparent header, aligned to the close button, with equal spacing now around the grid-password.

**Comment by travisdmathis** [comment]
Yep, will get the adjustments up today for iphone @luciorubeens

File: config.xml [hunk 1]
```diff
@@ -109,4 +109,6 @@
     <plugin name="cordova-plugin-splashscreen" spec="^5.0.2" />
     <plugin name="cordova-plugin-vibration" spec="^3.0.1" />
     <plugin name="cordova-plugin-screen-orientation" spec="~3.0.1" />
+    <allow-navigation href="http://10.0.0.7:8100" />
```
Comment (travisdmathis) [inline config.xml]: @luciorubeens why does cordova add this?

**Comment by travisdmathis** [comment]
@luciorubeens this should be ready to check on ios again.. looks a lot better on my galaxy :+1:

**Comment by luciorubeens** [comment]
Sorry my late reply, thank you! 👍

**Pull request merged**